### PR TITLE
refactor(node): reuse the initial difficulty as minimum difficulty an…

### PR DESCRIPTION
**Pull request by lrubiorod**
…d rename it

### CODE REVIEW

**Comment by tmpolaczyk** [comment]
Since this is just a refactor, can it wait until after the second hard fork is merged?

**Comment by lrubiorod** [comment]
> Since this is just a refactor, can it wait until after the second hard fork is merged?
It's ok for me

File: config/src/config.rs
Comment (tmpolaczyk) [inline config/src/config.rs:387]: Changing the names of the fields here can break some `witnet.toml`s. Not the ones used by most users only `witnet.toml`s used for testing which have a "consensus_constants" section. So it would be nice to show a warning if we parse a toml with the old name of the field instead of the new one, or something.

Comment (lrubiorod) [reply config/src/config.rs:387]: Due to this warning only be used for testing proposals, it could be done in a separated issue if it would be required

Comment (tmpolaczyk) [reply config/src/config.rs:387]: Ok, remind me to update my tomls after this is merged.

**Review by tmpolaczyk — APPROVED** [review]

**Pull request merged**
